### PR TITLE
Removed django.urls import fallback in wiki.models.

### PR DESCRIPTION
**Pull request by atombrella**

### CODE REVIEW

**Comment by rsalmaso** [comment]
@atombrella it seems it's the same in https://github.com/django-wiki/django-wiki/pull/795 (which is extract from https://github.com/django-wiki/django-wiki/pull/755) :)

File: src/wiki/models/__init__.py [hunk 1]
```diff
@@ -99,15 +101,9 @@ def reverse(*args, **kwargs):
...
-    shortcuts.reverse = reverse
-    urls.reverse_lazy = reverse_lazy
-except ImportError:
-    pass
+base.reverse = reverse
+base.reverse_lazy = reverse_lazy
+urls.reverse = reverse
+urls.reverse_lazy = reverse_lazy
+shortcuts.reverse = reverse
+urls.reverse_lazy = reverse_lazy
```
Comment (rsalmaso) [inline src/wiki/models/__init__.py]: dup of 107 (took me about 27 days to notice it :smile_cat: )

Comment (atombrella) [reply src/wiki/models/__init__.py]: OK. I just very quickly looked at the `ImportError`.

**Comment by atombrella** [comment]
No need for duplicate work. It's in #795 as pointed out.

**Pull request closed without merging**